new/LIT7346QeneLasabaya.xml

```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7346QeneLasabaya" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለሰብናየ፡ ቤተ፤ ንጉሥ፡ ሀልዮ፡ ሰርዌሁ፡ ዘኢይሰፈር፡ ምጠቁ።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">La-sabbǝnāya beta nǝguś hallǝyo sǝrwehu za-ʾi-yǝssaffar mǝṭaqu... 
+                    (qǝne of the kʷǝllǝkǝmu mawaddǝs-type)</title>
+                <title xml:lang="en" corresp="#t1">Concerning my scarf of the house, the king after he was bribing his army, who did not 
+                    distribute his elevated (?)...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne poem. The type is indicated as mawaddǝs in the manuscript. According to 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>, it is a 
+                    kʷǝllǝkǝmu mawaddǝs-type with nine verses. The author is not indicated.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes"/>
```

Comment:
I think we can safely exclude mawaddes from keywords, and leave only "Kwellekomu", if we thnk that the type is "Mawaddes Kwellekomu".